scale/job/models.py

```diff
@@ -3611,3 +3777,112 @@ class TaskUpdate(models.Model):
     class Meta(object):
         """Meta information for the database"""
         db_table = 'task_update'
+
+class JobTypeTagManager(models.Manager):
+    """Provides additional methods for handling job type tags
+    """
+
+    @transaction.atomic
+    def create_job_type_tags(self, job_type, tags):
+        """Creates a set of job type tags and saves them in the database. All database changes occur in an atomic
+        transaction.
+
+        :param job_type: The identifying name of the job type used by clients for queries
+        :type job_type: string
+        :param tag: The tags of the job type image
+        :type tag: list
+        :returns: The new job type tags
+        :rtype: [:class:`job.models.JobTypeTag`]
+        """
+        
+        job_type_tags = []
+        for tag in tags:
+            # Create the new job type tag
+            job_type_tag = JobTypeTag()
+            job_type_tag.job_type = job_type
+            job_type_tag.tag = tag
+            job_type_tags.append(job_type_tag)
+            
+        self.bulk_create(job_type_tags)
+
+        return job_type_tags
+
+    @transaction.atomic
+    def clear_job_type_tags(self, job_type):
```

Comment:
No need for either creation or removal of tags to be atomic for these functions.